Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -93,4 +131,69 @@ JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::symbol_species_getter)
     return vm.this_value(global_object);
 }
 
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::input_setter)
+{
+    auto new_value = vm.argument(0);
+    global_object.regexp_constructor()->m_input = new_value;
+    return new_value;
```

Comment:
This still needs to follow the spec - the one on GitHub we linked, not some random Microsoft docs. In this case:

https://github.com/tc39/proposal-regexp-legacy-features#set-regexpinput--val

So you'll have to implement the `SetLegacyRegExpStaticProperty` AO, which includes observable steps like throwing depending on the this value and calling ToString on the value being set.